cmd/common.go

```diff
@@ -139,6 +143,40 @@ func listRoleTemplateBindings(ctx *cli.Context, b []RoleTemplateBinding) error {
 	return writer.Err()
 }
 
+// getCurrentServer returns the name of the name and url of the current server in the local config
+func getCurrentServer(ctx *cli.Context) (string, string, error) {
+	cf, err := loadConfig(ctx)
+	if err != nil {
+		return "", "", err
+	}
+
+	serverName := cf.CurrentServer
+	return cf.CurrentServer, cf.Servers[serverName].URL, nil
+}
+
+func getCurrentKubeConfigForUser(ctx *cli.Context, user string) (*api.Config, error) {
+	cf, err := loadConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	kubeConfig, _ := cf.FocusedServer().KubeConfigs[fmt.Sprintf(kubeConfigKeyFormat, user, cf.FocusedServer().FocusedCluster())]
```

Comment:
nit: should `cf.FocusedServer()` be just one call that gets stored in a variable?